packages/quantic/force-app/main/default/lwc/quanticNotifications/quanticNotifications.html

```diff
@@ -12,6 +12,20 @@
           <div class="slds-notify__content slds-var-m-left_small">
             {notification.value}
           </div>
+          <div class="slds-notify__close">
+            <button
+              class="slds-button slds-button_icon slds-button_icon-inverse"
+              title="Close"
```

Comment:
This button needs a better title so when the screen reader reads it, the user understands really what this button is about.